examples/Tweet.hs

```diff
@@ -108,7 +97,10 @@ main = runBH' $ do
   let boost = Nothing
```

Comment:
Well, as it was before the change I think it is according to ES spec for  [Put Mapping case 2](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-put-mapping.html).
 That is, the mapping type is given by the URL and need not be specified in the JSON payload. However with my change of the mapping to work with the [PR #205 ] it is not really according to the manual, since the type name is defined twice. It works, but maybe not perfectly correct.